check_replication_queue.py

```diff
@@ -0,0 +1,39 @@
+import sys
+import waiting
+import click
+import logging
+import requests
+
+DEFAULT_TIMEOUT_THRESHOLD = 300
+DEFAULT_POLL_PERIOD = 60
+
+logging.basicConfig()
+logger = logging.getLogger("replication_queue")
+logger.setLevel(logging.INFO)
+
+
+def _is_replication_queue_empty(host: str, port: str):
+    response = requests.post(
+        url=f"http://{host}:{port}",
+        params={"query": "SELECT * FROM system.replication_queue"}
```

Comment:
On a general note, its not best practice to run SELECT * queries on OLAP databases. It is better to specify the columns you are interested in and look at their values.